### DISCUSSION TITLE
How much of the Zynq Z-7020 FPGA is used

### DISCUSSION THREAD

**Discussion by eiffelbebop** [open-sdr/openwifi#439] [Q&A]
I'm wondering how much of the available logic on the Zynq Z-7020 FPGA is currently used for the Openwifi design? I'm looking forward to trying some custom OFDM schemes, including MIMO. I'm wondering if there will be enough logic to add the extra TX and RX chains, or if the current design on the Z-7020 is already constrained by space?

**Comment by robgar2001** [comment]
Hi @eiffelbebop,

You can build the openwifi bitstream for the platform you like and have a look yourself at the utilization report, I don't know the numbers by heart.
